### PR TITLE
Release 3.4.2 build 1 with setuptools-git-versioning >=2.0, added to run

### PR DESCRIPTION
**Pull request by bobleesj**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bobleesj** [comment]
@conda-forge-admin, please rerender

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/diffpy.utils-fe
 
 Home: https://github.com/diffpy/diffpy.utils/
 
-Package license: [LICENSE](https://github.com/diffpy/diffpy.utils/blob/main/LICENSE.rst) and [LICENSE_DANSE](https://github.com/diffpy/diffpy.utils/blob/main/LICENSE_DANSE.txt)
+Package license: Free To Use But Restricted
```
Comment (bobleesj) [inline README.md:8]: Seems like it's automatically updated based on the meta.yaml file.

File: recipe/meta.yaml
Comment (bobleesj) [inline recipe/meta.yaml:1]: The only file that was modified manually.

**Comment by sbillinge** [comment]
lgtm

**Pull request merged**